tiledb/sm/c_api/tiledb_serialization.h

```diff
@@ -500,6 +500,89 @@ TILEDB_EXPORT int32_t tiledb_deserialize_config(
     int32_t client_side,
     tiledb_config_t** config) TILEDB_NOEXCEPT;
 
+/**
+ * Serializes the fragment info request into the given buffer.
+ *
+ * @note The caller must free the returned `tiledb_buffer_t`.
+ *
+ * @param ctx The TileDB context.
+ * @param fragment_info Fragment info to get the info to serialize.
+ * @param serialization_type Type of serialization to use.
+ * @param client_side Allows to specify different behavior depending on who is
+ * serializing, the client (1) or the Cloud server (0). This is sometimes needed
+ * since they are both using the same Core library APIs for serialization.
+ * @param buffer Will be set to a newly allocated buffer containing the
+ *      serialized fragment info request.
+ * @return `TILEDB_OK` for success and `TILEDB_ERR` for error.
+ */
+TILEDB_EXPORT int32_t tiledb_serialize_fragment_info_request(
```

Comment:
the `_request` doesn't match the naming schema for other serialization functions. Should this just be `tiledb_serialize_fragment_info`?